### PR TITLE
grml-setkeyboard: deprecate

### PR DESCRIPTION
**Pull request by zeha**
Point users towards grml-lang and dpkg-reconfigure keyboard-configuration instead.

Closes: grml/grml-scripts#10

### CODE REVIEW

File: usr_sbin/grml-config-root [hunk 1]
```diff
@@ -40,7 +40,7 @@ case $retval in
           exec grml-setlang
         fi
         if [ "$MENU" = "grml-setkeyboard" ]; then
-          exec grml-setkeyboard
+          exec dpkg-reconfigure keyboard-configuration
```
Comment (mika) [inline usr_sbin/grml-config-root:43]: Maybe I'm holding it wrong, but tried to test this inside a VM by choosing `German (Austria)` keyboard, but doesn't seem to end up with such a keyboard setting then (tested inside tty1`), not claiming that my environment might be the problem, but I'd like further verification/testing yet before shipping that :)

Comment (zeha) [reply usr_sbin/grml-config-root]: Thanks for testing :-/

Apparently console-setup doesn't bother to apply the new settings after debconf exits. I'll spare you the details 😱

File: usr_sbin/grml-setkeyboard [hunk 2]
```diff
@@ -7,120 +7,9 @@
...
-    (0)
-          # everything ok
-          LANG=C $DIALOG --stdout --msgbox "Writing keyboard settings ($KEYTABLE) to /etc/sysconfig/keyboard was successful." 0 0
-          ;;
-    *)
-          LANG=C $DIALOG --stdout --msgbox "Error writing settings for $KEYTABLE to /etc/sysconfig/keyboard." 0 0
-          ;;
-esac
+echo "${PN}: this program is deprecated."
+echo "${PN}: please use grml-lang instead."
```
Comment (mika) [inline usr_sbin/grml-setkeyboard:12]: I guess we should redirect the error message to stderr (via `>&2`)

**Review by mika — COMMENTED** [review]
Wasn't aware of this console-setup.service restart behavor, TIL :)

File: usr_sbin/grml-setkeyboard (hunk 2 above)
Comment (zeha) [reply usr_sbin/grml-setkeyboard]: Did that now.

**Pull request merged**